src/app/(nextjs_migration)/(authenticated)/user/welcome/details/page.tsx

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image from "next/image";
+import Script from "next/script";
+
+import { getL10n } from "../../../../../functions/server/l10n";
+import { getLocale } from "../../../../../../utils/fluent.js";
+import { authOptions } from "../../../../../api/utils/auth";
+
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: "Welcome - Details", // FIXME l10n.getString("breach-meta-title"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: process.env.SERVER_URL,
+      images: ["/images/og-image.webp"],
+    },
+  };
+}
+
+export default async function UserBreaches() {
+  const l10n = getL10n();
+
+  return (
+    <>
+      <main>
+        <h1>Enter the details you want to protect</h1>
+        <p>
+          We’ll use this to find exposures of your personal information, and
+          then guide you step-by-step on how to fix it.
+          <a>Why do we need this info?</a>
+        </p>
+
+        <section>
+          <form method="post" action="scanning">
+            <div>
+              <label for="name">First name: </label>
+              <input type="text" name="firstname" id="firstname"></input>
```

Comment:
W3c validator didn't like these `</input>` tags.

```
Error: Stray end tag input.

[From line 23, column 66; to line 23, column 73](https://validator.w3.org/nu/#l23c73)

itystate"></input>↩
```

Maybe this instead?

```suggestion
              <input type="text" name="firstname" id="firstname" />
```

Reply:
Huh, I'm surprised to hear that actually... will take a look.

Reply:
A trailing slash is not needed either. Just write:

`<input type="text" name="firstname" id="firstname">`

It's not an error to use a trailing slash but the W3C validator gives you the following information if you use it:

> Trailing slash on void elements has no effect and interacts badly with unquoted attribute values.

Reply:
Hm, our JSX linter wants a closing tag here, I'll do that for now. Note that the final HTML output might look slightly different! JSX is XML-ish.

Reply:
Yeah, it's not actually HTML, it's roughly syntactic sugar for programmatically creating DOM elements. Running the W3C validator on the input source won't help much - I imagine it doesn't like attributes like `className` or `htmlFor` either :sweat_smile: